terminal.cpp

```diff
@@ -1112,7 +1112,7 @@ void Terminal::escControlChar(const QString& seq)
             iTabStops.append(QList<int>());
 
         iTabStops[cursorPos().y()-1].append(cursorPos().x());
-        qSort(iTabStops[cursorPos().y()-1]);
+        std::sort(iTabStops[cursorPos().y()-1].begin(), iTabStops[cursorPos().y()-1].end());
```

Comment:
I wonder if there is any possibility that these iterators from `begin()` and `end()` can point to different `QList` instances. Although I think the answer is probably no, and I hope it is no, it's not that obvious from documentation. It might be safer to take `const QList<int> &` first and then `begin()` and `end()` from that.

https://doc.qt.io/archives/qt-5.6/qlist.html#operator-5b-5d

Reply:
Shouldn't be much of chance for the same index to change content between creating the iterators, but indeed this was at least messy to read. Adjusted.